snowflake_blueprints/upload_file.py

```diff
@@ -171,21 +259,31 @@ def upload_data_with_put(source_full_path,
     if insert_method == 'replace':
         snowflake_results = execute_drop_command(
             db_connection, table_name, snowflake_results)
-        create_table(
-            source_full_path,
-            table_name,
-            insert_method,
-            db_connection)
-        snowflake_results = execute_put_command(
-            db_connection, parquet_path, table_name, snowflake_results)
-        snowflake_results = execute_copyinto_command(
-            db_connection, table_name, snowflake_results)
+        if snowflake_data_types is not None:
```

Comment:
Quick fix here.

When a user doesn't provide a value in the UI, it sends an empty string. That will not be none, so it will try to create a table but won't have fields to parse.

I don't remember how we account for that in other Blueprints... I think it's by making the argument have a default of empty string and building all logic in the script around that.

Didn't test, but pretty sure.